packages/ui/v2/modules/event-types/MultiDropdownSelect.tsx

```diff
@@ -0,0 +1,113 @@
+import autoAnimate from "@formkit/auto-animate";
+import React, { useEffect, useRef } from "react";
+import { components, GroupBase, Props, ValueContainerProps } from "react-select";
+
+import { Icon } from "@calcom/ui/Icon";
+
+// import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Select } from "../..";
+
+const LimitedChipsContainer = <Option, IsMulti extends boolean, Group extends GroupBase<Option>>({
+  children,
+  ...props
+}: ValueContainerProps<Option, IsMulti, Group>) => {
+  if (!props.hasValue) {
+    return <components.ValueContainer {...props}>{children as React.ReactNode[]}</components.ValueContainer>;
+  }
+  const CHIPS_LIMIT = 2;
+  // TODO:: fix the following ts error
+  // @ts-expect-error: @see children is an array but identified as object resulting in the error
+  const [chips, other] = children;
+  const overflowCounter = chips.slice(CHIPS_LIMIT).length;
+  const displayChips = chips.slice(overflowCounter, overflowCounter + CHIPS_LIMIT);
+  // console.log(props.getValue(), props.clearValue);
```

Comment:
Is there any reason to keep this console.log?

Reply:
Oh, that's a mistake. No, we don't need it. I was debugging the type errors and somehow missed on removing it entirely